src/hosting/api.ts

```diff
@@ -373,6 +373,27 @@ export async function addAuthDomain(project: string, url: string): Promise<strin
   return await updateAuthDomains(project, authDomains);
 }
 
+/**
+ * Removes channel domain from Firebase Auth list.
+ * @param project the project ID.
+ * @param url the url of the channel.
+ */
+export async function removeAuthDomain(project: string, url: string): Promise<string[]> {
+  const domains = await getAuthDomains(project);
+  if (domains.length < 1) {
+    return domains;
+  }
+  const targetDomain = url.replace("https://", "");
+  const authDomains: string[] = [];
+  domains.forEach((domain: string) => {
+    if (domain == targetDomain) {
+      return;
+    }
+    authDomains.push(domain);
+  });
```

Comment:
```typescript
const authDomains = domains.forEach((d: string) => d != targetDomain);
```

(untested, but I believe that passes lint rules)

Reply:
I think you meant `.filter`